Userland/Libraries/LibCrypto/Hash/PBKDF2.h

```diff
@@ -77,7 +77,7 @@ class PBKDF2 {
         }
 
         // 5. Output the derived key DK
-        return key;
+        return move(key);
```

Comment:
A `move` for a return value should almost never be necessary

Reply:
And that's also what CI complains about. You're right, thanks :)